test/compile/scale/domain.test.ts

```diff
@@ -235,18 +235,15 @@ describe('compile/scale', () => {
 
         it('should return the correct domain for yearmonth T when specify sort',
           function() {
+            const sortDef = {op: 'mean', field: 'precipitation', order: 'descending'} ;
```

Comment:
instead of the as below, write `sortDef: VgSortField` here. But it shouldn't really be necessary.